plugins/jobtype/jobtypes/teradataToHdfs/plugin.properties

```diff
@@ -0,0 +1,5 @@
+mr.listener.visualizer=false
+
+#Below is to use latest avro jar for TDCH
```

Comment:
I think reusing this jar will require changing build script either in azkaban-plugins or azkaban
https://github.com/azkaban/azkaban-plugins/blob/master/build.xml
https://github.com/azkaban/azkaban/blob/master/build.gradle#L352

Let us sync up offline to discuss this.